<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk+" version="2.16"/>
  <!-- interface-naming-policy project-wide -->
  <object class="GtkDialog" id="dbwindow">
    <property name="border_width">5</property>
    <property name="title" translatable="yes">Pigeon Planner database tool</property>
    <property name="icon">../images/icon_logo.png</property>
    <property name="type_hint">normal</property>
    <signal name="delete_event" handler="quit_program"/>
    <child internal-child="vbox">
      <object class="GtkVBox" id="dialog-vbox1">
        <property name="visible">True</property>
        <property name="spacing">2</property>
        <child>
          <object class="GtkNotebook" id="notebook1">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <child>
              <object class="GtkAlignment" id="alignment1">
                <property name="visible">True</property>
                <property name="top_padding">4</property>
                <property name="bottom_padding">4</property>
                <property name="left_padding">4</property>
                <property name="right_padding">4</property>
                <child>
                  <object class="GtkTable" id="table1">
                    <property name="visible">True</property>
                    <property name="n_rows">3</property>
                    <property name="n_columns">3</property>
                    <property name="column_spacing">8</property>
                    <property name="row_spacing">12</property>
                    <child>
                      <object class="GtkButton" id="button_optimize">
                        <property name="label" translatable="yes">Optimize</property>
                        <property name="visible">True</property>
                        <property name="can_focus">True</property>
                        <property name="receives_default">True</property>
                        <property name="image">image3</property>
                        <property name="yalign">0</property>
                        <property name="image_position">top</property>
                        <signal name="clicked" handler="on_button_optimize_clicked"/>
                      </object>
                      <packing>
                        <property name="x_options"></property>
                        <property name="y_options"></property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkButton" id="button_check">
                        <property name="label" translatable="yes">Check</property>
                        <property name="visible">True</property>
                        <property name="can_focus">True</property>
                        <property name="receives_default">True</property>
                        <property name="image">image2</property>
                        <property name="yalign">0</property>
                        <property name="image_position">top</property>
                        <signal name="clicked" handler="on_button_check_clicked"/>
                      </object>
                      <packing>
                        <property name="top_attach">1</property>
                        <property name="bottom_attach">2</property>
                        <property name="x_options"></property>
                        <property name="y_options"></property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkButton" id="button_delete">
                        <property name="label">gtk-delete</property>
                        <property name="visible">True</property>
                        <property name="can_focus">True</property>
                        <property name="receives_default">True</property>
                        <property name="use_stock">True</property>
                        <property name="yalign">0</property>
                        <property name="image_position">top</property>
                        <signal name="clicked" handler="on_button_delete_clicked"/>
                      </object>
                      <packing>
                        <property name="top_attach">2</property>
                        <property name="bottom_attach">3</property>
                        <property name="x_options"></property>
                        <property name="y_options"></property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="label8">
                        <property name="visible">True</property>
                        <property name="xalign">0</property>
                        <property name="yalign">0</property>
                        <property name="label" translatable="yes">Optimze the database tables. This will defragment the data and free unused space.</property>
                        <property name="wrap">True</property>
                      </object>
                      <packing>
                        <property name="left_attach">1</property>
                        <property name="right_attach">2</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="label9">
                        <property name="visible">True</property>
                        <property name="xalign">0</property>
                        <property name="yalign">0</property>
                        <property name="label" translatable="yes">Check the database for errors and try to repair them.</property>
                        <property name="wrap">True</property>
                      </object>
                      <packing>
                        <property name="left_attach">1</property>
                        <property name="right_attach">2</property>
                        <property name="top_attach">1</property>
                        <property name="bottom_attach">2</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="label10">
                        <property name="visible">True</property>
                        <property name="xalign">0</property>
                        <property name="yalign">0</property>
                        <property name="label" translatable="yes">Delete the entire database from your hard drive. You will loose all pigeons, results and other data.</property>
                        <property name="wrap">True</property>
                      </object>
                      <packing>
                        <property name="left_attach">1</property>
                        <property name="right_attach">2</property>
                        <property name="top_attach">2</property>
                        <property name="bottom_attach">3</property>
                        <property name="y_options">GTK_FILL</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkImage" id="image_optimize">
                        <property name="visible">True</property>
                      </object>
                      <packing>
                        <property name="left_attach">2</property>
                        <property name="right_attach">3</property>
                        <property name="x_options"></property>
                        <property name="y_options"></property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkImage" id="image_check">
                        <property name="visible">True</property>
                      </object>
                      <packing>
                        <property name="left_attach">2</property>
                        <property name="right_attach">3</property>
                        <property name="top_attach">1</property>
                        <property name="bottom_attach">2</property>
                        <property name="x_options"></property>
                        <property name="y_options"></property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkImage" id="image_delete">
                        <property name="visible">True</property>
                      </object>
                      <packing>
                        <property name="left_attach">2</property>
                        <property name="right_attach">3</property>
                        <property name="top_attach">2</property>
                        <property name="bottom_attach">3</property>
                        <property name="x_options"></property>
                        <property name="y_options"></property>
                      </packing>
                    </child>
                  </object>
                </child>
              </object>
            </child>
            <child type="tab">
              <object class="GtkLabel" id="label1">
                <property name="visible">True</property>
                <property name="label" translatable="yes">General</property>
              </object>
              <packing>
                <property name="tab_fill">False</property>
              </packing>
            </child>
            <child>
              <object class="GtkVBox" id="vbox2">
                <property name="visible">True</property>
                <property name="spacing">4</property>
                <child>
                  <object class="GtkHBox" id="hbox1">
                    <property name="visible">True</property>
                    <property name="spacing">8</property>
                    <child>
                      <object class="GtkImage" id="image1">
                        <property name="visible">True</property>
                        <property name="stock">gtk-dialog-warning</property>
                        <property name="icon-size">6</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">False</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="label3">
                        <property name="visible">True</property>
                        <property name="yalign">0</property>
                        <property name="label" translatable="yes">Warning:</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">False</property>
                        <property name="position">1</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="label4">
                        <property name="visible">True</property>
                        <property name="xalign">0</property>
                        <property name="yalign">0</property>
                        <property name="label" translatable="yes">Only execute SQL statements if you know what you are doing! Entering invalid or wrong statements may cause data loss!</property>
                        <property name="wrap">True</property>
                      </object>
                      <packing>
                        <property name="position">2</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">0</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkScrolledWindow" id="scrolledwindow1">
                    <property name="visible">True</property>
                    <property name="can_focus">True</property>
                    <property name="hscrollbar_policy">automatic</property>
                    <property name="vscrollbar_policy">automatic</property>
                    <property name="shadow_type">in</property>
                    <child>
                      <object class="GtkTextView" id="textview_sql">
                        <property name="visible">True</property>
                        <property name="can_focus">True</property>
                        <property name="wrap_mode">word</property>
                      </object>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkHButtonBox" id="hbuttonbox1">
                    <property name="visible">True</property>
                    <child>
                      <object class="GtkButton" id="button_execute">
                        <property name="label">gtk-execute</property>
                        <property name="visible">True</property>
                        <property name="sensitive">False</property>
                        <property name="can_focus">True</property>
                        <property name="receives_default">True</property>
                        <property name="use_stock">True</property>
                        <signal name="clicked" handler="on_button_execute_clicked"/>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">False</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkButton" id="button_clear">
                        <property name="label">gtk-clear</property>
                        <property name="visible">True</property>
                        <property name="can_focus">True</property>
                        <property name="receives_default">True</property>
                        <property name="use_stock">True</property>
                        <signal name="clicked" handler="on_button_clear_clicked"/>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">False</property>
                        <property name="position">1</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="expand">False</property>
                    <property name="fill">False</property>
                    <property name="position">2</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkScrolledWindow" id="scrolledwindow2">
                    <property name="visible">True</property>
                    <property name="can_focus">True</property>
                    <property name="hscrollbar_policy">automatic</property>
                    <property name="vscrollbar_policy">automatic</property>
                    <property name="shadow_type">in</property>
                    <child>
                      <object class="GtkTextView" id="textview_output">
                        <property name="visible">True</property>
                        <property name="can_focus">True</property>
                        <property name="editable">False</property>
                        <property name="wrap_mode">word</property>
                      </object>
                    </child>
                  </object>
                  <packing>
                    <property name="position">3</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="position">1</property>
              </packing>
            </child>
            <child type="tab">
              <object class="GtkLabel" id="label2">
                <property name="visible">True</property>
                <property name="label" translatable="yes">SQL</property>
              </object>
              <packing>
                <property name="position">1</property>
                <property name="tab_fill">False</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="position">1</property>
          </packing>
        </child>
        <child internal-child="action_area">
          <object class="GtkHButtonBox" id="dialog-action_area1">
            <property name="visible">True</property>
            <property name="layout_style">edge</property>
            <child>
              <object class="GtkButton" id="button_log">
                <property name="label" translatable="yes">Logfile</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="image">image4</property>
                <signal name="clicked" handler="on_button_log_clicked"/>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">False</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="button_quit">
                <property name="label">gtk-quit</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="use_stock">True</property>
                <signal name="clicked" handler="quit_program"/>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">False</property>
                <property name="position">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="pack_type">end</property>
            <property name="position">0</property>
          </packing>
        </child>
      </object>
    </child>
    <action-widgets>
      <action-widget response="0">button_log</action-widget>
      <action-widget response="0">button_quit</action-widget>
    </action-widgets>
  </object>
  <object class="GtkImage" id="image4">
    <property name="visible">True</property>
    <property name="stock">gtk-file</property>
  </object>
  <object class="GtkImage" id="image3">
    <property name="visible">True</property>
    <property name="stock">gtk-properties</property>
  </object>
  <object class="GtkImage" id="image2">
    <property name="visible">True</property>
    <property name="stock">gtk-find</property>
  </object>
  <object class="GtkSizeGroup" id="sizegroup1">
    <widgets>
      <widget name="button_optimize"/>
      <widget name="button_check"/>
      <widget name="button_delete"/>
    </widgets>
  </object>
</interface>
